src/SFA.DAS.EmployerApprenticeshipsService.Domain/Data/Entities/Account/Account.cs

```diff
@@ -38,5 +41,12 @@ private void RequiresTransferConnectionInvitationDoesNotAlreadyExist(Account rec
             if (anyTransferConnectionInvitations)
                 throw new Exception("Requires transfer connection invitation does not already exist.");
         }
+
+        private void RequiresReceiverAccountIsNotAlreadyASender(Account receiverAccount)
+        {
+            var receiverIsASender = receiverAccount.SentTransferConnectionInvitations.Any(i => i.Status != TransferConnectionInvitationStatus.Rejected);
```

Comment:
This should probably be encapsulated by a method on `Account`.

Reply:
'tis done